### ISSUE TITLE
Bug - Topology - Cola animation makes page crash on layout change

### ISSUE THREAD

**Issue by jpinsonneau** [patternfly/patternfly-react#8659]
**Describe the problem**
A clear and concise description of the problem. Which components are affected?
On topology component, the page crash if you try to change layout while the animation is still running.

**How do you reproduce the problem?**
Provide steps to reproduce. A [codesandbox](https://codesandbox.io/s/serverless-cherry-q9t3f ) demonstrating the problem is appreciated.

Open https://topology-styles.surge.sh/
Move on Topology Demo -> Topology Package
Set nodes to 300, edges to 100 and groups to 0
Apply and change layout to Force while animation is still running

The console show the following exception:
```
main.85030db1f765c8a181f7.bundle.js:84 Error: GraphElement with ID 'g1' has no controller.
    at n8.getController (main.85030db1f765c8a181f7.bundle.js:118:780409)
    at main.85030db1f765c8a181f7.bundle.js:118:814042
    at main.85030db1f765c8a181f7.bundle.js:118:803882
    at main.85030db1f765c8a181f7.bundle.js:118:803016
    at ae (main.85030db1f765c8a181f7.bundle.js:15:5348)
    at e.track (main.85030db1f765c8a181f7.bundle.js:15:16243)
    at main.85030db1f765c8a181f7.bundle.js:118:802992
    at main.85030db1f765c8a181f7.bundle.js:118:802825
    at S8 (main.85030db1f765c8a181f7.bundle.js:118:802966)
    at l (main.85030db1f765c8a181f7.bundle.js:118:803860)
oa @ main.85030db1f765c8a181f7.bundle.js:84
main.85030db1f765c8a181f7.bundle.js:15 [mobx] Encountered an uncaught exception that was thrown by a reaction or observer component, in: 'Reaction[observer]' Error: GraphElement with ID 'g1' has no controller.
    at n8.getController (main.85030db1f765c8a181f7.bundle.js:118:780409)
    at main.85030db1f765c8a181f7.bundle.js:118:814042
    at main.85030db1f765c8a181f7.bundle.js:118:803882
    at main.85030db1f765c8a181f7.bundle.js:118:803016
    at ae (main.85030db1f765c8a181f7.bundle.js:15:5348)
    at e.track (main.85030db1f765c8a181f7.bundle.js:15:16243)
    at main.85030db1f765c8a181f7.bundle.js:118:802992
    at main.85030db1f765c8a181f7.bundle.js:118:802825
    at S8 (main.85030db1f765c8a181f7.bundle.js:118:802966)
    at l (main.85030db1f765c8a181f7.bundle.js:118:803860)
e.reportExceptionInDerivation @ main.85030db1f765c8a181f7.bundle.js:15
topology-demo-nav-link:1          Failed to load resource: the server responded with a status of 404 (Not Found)
```

**Expected behavior**
A clear and concise description of the expected behavior.
The animation should completely stop when layout is changed

**Is this issue blocking you?**
List the workaround if there is one.
None

**Screenshots**
If applicable, add screenshots to help explain the issue.
![image](https://user-images.githubusercontent.com/91894519/217241066-f90d0a7f-bcbf-4e09-8cca-5e1e787ec36e.png)
![image](https://user-images.githubusercontent.com/91894519/217241100-382e46e0-96a9-4f97-848d-0bbd233bacaf.png)

**What is your environment?**
 - OS: fedora
 - Browser: chrome
 - Version: 109.0.5414.119

**What is your product and what release date are you targeting?**
Red Hat - Network Observability
Already in production

**Any other information?**
Maybe related to https://github.com/patternfly/patternfly-react/issues/8638

**Comment by jpinsonneau** [comment]
Looking at the stack trace debuging our product I found that `action` calling `update` on every node:
https://github.com/patternfly/patternfly-react/blob/e264cf1dde47e46486e4049f3f58cadfab85f900/packages/react-topology/src/layouts/ColaLayout.ts#L66

**Comment by jpinsonneau** [comment]
Solved in `@patternfly/react-topology@4.91.38`
